src/xwayland.c

```diff
@@ -32,6 +34,17 @@ xwayland_surface_contains_window_type(
 	return false;
 }
 
+static bool
+view_contains_xwayland_window_type(struct view *view, int32_t window_type)
```

Comment:
Sugget changing to `xwayland_view_contains_window_type` or something like that, functions starting with `view_` should usually only be defined in `src/view.c`.